topdown/graphql.go

```diff
@@ -5,32 +5,19 @@
 package topdown
 
 import (
+	"encoding/json"
 	"fmt"
 	"strings"
 
-	gqltop "github.com/vektah/gqlparser/v2"
 	gqlast "github.com/vektah/gqlparser/v2/ast"
```

Comment:
These need updating with the internalized code of #5067.

Reply:
This has been rolled into my most recent batch of changes. :smile: